python/Ganga/Utility/execute.py

```diff
@@ -230,10 +228,10 @@ def execute(command,
 
     if update_env:
         env_output_key = 'env_output'
-        update_env_thread = update_thread(env_file_pipes, thread_output, env_output_key)
+        update_env_thread = update_thread(env_file_pipes, thread_output, env_output_key, require_output=True)
     if not shell:
         pkl_output_key = 'pkl_output'
-        update_pkl_thread = update_thread(pkl_file_pipes, thread_output, pkl_output_key)
+        update_pkl_thread = update_thread(pkl_file_pipes, thread_output, pkl_output_key, require_output=False)
```

Comment:
OK, in the case we expect output we explicitly check that we get some.

I assumed the case of `require_output=True` was the output is allowed but not needed and that `require_output=False` was that no output is expected.
I'm stuck at thinking of a better parameter name however...

Are there any situations where we are currently getting errors due to not expecting any feedback via the streams if we're executing a python command?